Feverup/decap-cms
# PNW-1375 - Added stale workflow

## 📱 Which kind of feature is this PR?

Add `stale` state in order to avoid change to be removed

## ℹ️ What's this PR do?

- Added `stale` state to the workflow in a change
<img width="1728" alt="Screenshot 2023-06-26 at 17 14 43" src="https://github.com/Feverup/decap-cms/assets/59401547/b1c4b375-4cc9-49e5-a2c2-158113f0ee71">

- Added `stale` state to the workflows page
<img width="1728" alt="Screenshot 2023-06-26 at 17 14 29" src="https://github.com/Feverup/decap-cms/assets/59401547/8c90a83f-aa1f-44f8-9238-5ac6c6cc8fd3">

## 📱 How should this be tested?
1. Setup `landings-lib` to run the build of this PR by running:
- Run a `yarn build` 
- Copy the output inside `packages/netlify-cms/dist/netlify-cms.js` to `landings-lib` on `static/js`
3. Create a change on the CMS
4. Go to the PR and update the `labels` to `netlify-cms/stale`

- [x] Check that the `stale` state appears correctly on the workflow
- [x] Check that the `stale` state has a red color and it dosn't display on the workflow states
- [x] Check that when moving a change on the workflow page to `stale` it gives an error
